setup/so-functions

```diff
@@ -1954,6 +1962,8 @@ saltify() {
 			exit 1
 		fi
 
+		retry 50 10 "apt-get update" >> "$setup_log" 2>&1 || exit 1
```

Comment:
Shouldn't you do the update before the upgrade?

Reply:
Removing line per other conversation.